src/zeit/push/interfaces.py

```diff
@@ -244,6 +247,34 @@ class MobileButtonsSource(zeit.cms.content.sources.XMLSource):
 MOBILE_BUTTONS_SOURCE = MobileButtonsSource()
 
 
+class PayloadTemplateSource(zc.sourcefactory.basic.BasicSourceFactory):
+
+    # For testing purposes, this should not be cached at all
+    # @CONFIG_CACHE.cache_on_arguments()
```

Comment:
Wie evtl. schonmal gesagt, wenn die Werte der Source ICMSContent-Objekte sind, kommen die ja schon ausm DAV-Cache, d.h. da nochmal in der Source zu cachen ist glaube ich überflüssig.